src/App.js

```diff
@@ -1,41 +1,29 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import Home from "./routes/Home";
```

Comment:
Suggestion : you can make barrel export for images where you export all the images in one file and import it from one file at here.

P.S: You can do barrel export for components ,pages, contexts etc..